bower.json

```diff
@@ -2,6 +2,10 @@
   "name": "westtowest",
   "private": true,
   "dependencies": {
-    "jquery": "~2.1.4"
+    "jquery": "~2.1.4",
+    "slider-pro": "^1.2.2"
+  },
+  "resolutions": {
+    "promise": "^7.1.1"
```

Comment:
yes, although it's not explicitly stated anywhere related to the project, it arose from installing the plugin through bower.